docs/tempo/website/api_docs/_index.md

```diff
@@ -83,12 +83,12 @@ Agent / OpenTelemetry Collector / Jaeger Agent.
 |  Jaeger | GRPC | [Link](https://www.jaegertracing.io/docs/latest/apis/#span-reporting-apis) |
 |  Zipkin | HTTP | [Link](https://zipkin.io/zipkin-api/) |
 
-_For information on how to use the Zipkin endpoint with curl (for debugging purposes) check [here](pushing-spans-with-http)._ 
+For information on how to use the Zipkin endpoint with curl (for debugging purposes) check [here](pushing-spans-with-http). 
 
 ### Query
 
-Tempo's Query API is simple. The following request is used to retrieve a trace from the query frontend service in 
-a microservices deployment, or the Tempo endpoint in a monolithic mode deployment.
+The following request is used to retrieve a trace from the query frontend service in 
```

Comment:
Last I heard we wanted everything to be "monolithic mode". Although I preferred "single binary" myself.

Reply:
@joe-elliott Which term is the correct terminology to use? I'm happy to change it to monolithic mode and then be consistent with it.

Reply:
I will update it. Thank you.